src/main/java/com/faforever/client/game/GameService.java

```diff
@@ -631,4 +670,19 @@ private void removeGame(GameInfoMessage gameInfoMessage) {
       uidToGameInfoBean.remove(gameInfoMessage.getUid());
     }
   }
+
+  @EventListener
+  public void onDiscordGameJoinEvent(DiscordJoinEvent discordJoinEvent) {
+    Integer gameId = discordJoinEvent.getGameId();
+    Game game = getByUid(gameId);
+    boolean disallowJoinsViaDiscord = preferencesService.getPreferences().isDisallowJoinsViaDiscord();
+    if (disallowJoinsViaDiscord) {
+      log.debug("Join was requested via Discord but was rejected due to it being disabled in settings");
+      return;
+    }
+    if (game == null) {
+      throw new IllegalStateException(MessageFormat.format("Could not find game to join, with id: {0}", discordJoinEvent.getGameId()));
```

Comment:
```
System.out.println(MessageFormat.format("{0}", 11111111));
```
results in:
```
11,111,111
```

Therefore, either don't use MessageFormat (instead use StringFormat) or use `{0,number,#}` to prevent the formatting. As a rule of thumb, whenever the message should be displayed to the user, use `MessageFormat`, if it's technical only, use `StringFormat`.